website/routes.py

```diff
@@ -337,6 +337,17 @@ def make_url_map(app):
             json_renderer,
         )
     ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/confirmed_emails/', 'get', auth_views.confirm_user_get, json_renderer),
+        Rule('/dashboard/confirmed_emails/', 'put', auth_views.add_confirmed_email, json_renderer)
```

Comment:
Rather than having 2 duplicate rules, you can pass a list of supported methods, e.g. `['get', 'put']`.